File: quasar/Quasar.DataTransfer/Usuarios/Requests/ClienteInserirRequest.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Quasar.DataTransfer.Usuarios.Requests
{
    public class ClienteInserirRequest
    {
        public string Nome { get; set; }
        public string Sobrenome { get; set; }
        public string? NomeFantasia { get; set; }
        public string CpfCnpj { get; set; }
        public string? InscricaoEstadual { get; set; }
        public string? RazaoSocial { get; set; }
        public int TipoCliente { get; set; }
    }
}

File: quasar-front/src/app/usuario/models/request/clienteInserirRequest.ts
export class ClienteInserirRequest {
  public nome: string;
  public sobrenome: string;
  public nomeFantasia: string;
  public cpfCnpj: string;
  public inscricaoEstadual: string;
  public razaoSocial: string;
  public tipoCliente: string;

  constructor(params: Partial<ClienteInserirRequest>) {
    this.nome = params.nome || '';
    this.sobrenome = params.sobrenome || '';
    this.nomeFantasia = params.nomeFantasia || '';
    this.cpfCnpj = params.cpfCnpj || '';
    this.inscricaoEstadual = params.inscricaoEstadual || '';
    this.razaoSocial = params.razaoSocial || '';
    this.tipoCliente = params.tipoCliente || '';
  }
}
